packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -454,6 +470,11 @@ export enum CompressionAlgorithms {
 	lz4 = "lz4",
 }
 
+/**
+ * Default count of reserved Ids that the runtime IdCompressor will initialize itself with if enabled.
+ */
+export const defaultIdCompressorReservedIdCount = 10;
```

Comment:
It's very hard to say what changes if this number changes and how devs should evaluate changes here.
Also is it up to runtime to decide? I.e. do we expect different users of compressor library starting with different values? Why?

Reply:
This reserves a fixed number of positive integers (final IDs) for use immediately, meaning it essentially gets you N IDs without waiting for an ack. It does it by having all clients agree to reserve that many initially; this has the downside of never really being able to change it without a code migration. It's not even a feature I'm sure we would use, and it definitely has to be a decision made by the container runtime and never exposed publicly.